image_width: 960
image_height: 540
camera_name: left
camera_matrix:
  rows: 3
  cols: 3
  data: [794.1614027044714, 0, 422.5932859632485, 0, 796.0641260589528, 271.551956769938, 0, 0, 1]
distortion_model: plumb_bob
distortion_coefficients:
  rows: 1
  cols: 5
  data: [-0.4237276970418096, 0.2119590537707229, -0.005342448349061238, -0.00279702554668938, 0]
rectification_matrix:
  rows: 3
  cols: 3
  data: [0.9918430516848367, -0.00691989307535249, -0.1272771617547165, 0.007502272722070515, 0.9999634651504484, 0.004096860787445452, 0.1272441618641667, -0.005018310884522798, 0.9918587297732233]
projection_matrix:
  rows: 3
  cols: 4
  data: [805.385162848315, 0, 614.1706352233887, 0, 0, 805.385162848315, 266.1567878723145, 0, 0, 0, 1, 0]